app/assets/javascripts/users.js

```diff
@@ -0,0 +1,25 @@
+function getRepositories() {
+  var url = $("#repositories").data("repositories")
+  $.getJSON(url, function(data) {
+    $.each(data, function(_idx, repository) {
+      $("#repositories").append("<tr>")
+      $("#repositories").append("<td>" + repository.name + "</td>")
+      $("#repositories").append("<td><a href=\"" + repository.html_url + "\">Go</a></td>")
+      $("#repositories").append("<td>" + repository.stargazers_count + "</td>")
+      $("#repositories").append("<td>" + repository.forks + "</td>")
+      $("#repositories").append("</tr>")
+    })
+  })
+}
+
+function getTemperature() {
+  var url = $("#temperature").data("temp")
```

Comment:
Maybe you can pass the URL in other way instead of fetching it straight from the DOM